python/metatensor_torch/metatensor/torch/atomistic/_extensions.py

```diff
@@ -58,11 +58,49 @@ def _featomic_deps_path():
     return deps_path
 
 
+def _sphericart_deps_path():
+    import sphericart
+
+    deps_path = [sphericart._c_lib._lib_path()]
```

Comment:
This is not needed for sphericart, since the same implementation is duplicated in `libsphericart` and `libsphericart_torch`.